python/nano/test/tf/keras/test_embeddings.py

```diff
@@ -24,7 +24,7 @@ def test_embedding_gradient_sparse():
     from bigdl.nano.tf.keras import Sequential
     import tensorflow as tf
     model = Sequential()
-    model.add(Embedding(1000, 64, input_length=10, regularizer=tf.keras.regularizers.L2()))
+    model.add(Embedding(1000, 64, input_length=10, activity_regularizer=tf.keras.regularizers.L2()))
```

Comment:
I think you should test `embeddings_regularizer` here, otherwise, it won't run into your updated logic.

Reply:
Ok.